week-2/debug/1.js

```diff
@@ -1,8 +1,9 @@
 // Predict and explain first...
 
 function sum(a, b) {
-  return;
-  a + b;
+  return (a + b);
 }
 
 console.log(`The sum of 10 and 32 is ${sum(10, 32)}`);
+
+//*It will undefined because return needs braces()
```

Comment:
This is slightly wrong I'm afraid.  the return;  on line 4 in the orginal makes the function return before it runs line 5 Brackets have nothing to do with it.